UIKit_DZ/UIKit_DZ/CustomView/ChekBoxSwitch.swift

```diff
@@ -0,0 +1,34 @@
+// ChekBoxSwitch.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+///  Вьюшка с ценами и свитчом
+final class ChekBoxSwitch: UIView {
+    lazy var nubmerPrice = 0
+    lazy var title = UILabel()
```

Comment:
```suggestion
    var title = UILabel()
```
зачем тут везде Лейзи?